package.json

```diff
@@ -31,14 +31,15 @@
     "node": ">=14"
   },
   "dependencies": {
+    "@openzeppelin/contracts-v4": "npm:@openzeppelin/contracts@4.7.0",
```

Comment:
i htink i'd rather this just be normal like it used to be, and then create a new mapping for the erc721 to solve whatever the problem there was?